test.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
For now this script will work only on CI, not yet ready to be run locally for testing. Separately, I'll create a guide how to setup and run tests locally, which is quite simple to be honest.